packages/example/src/screens/CheckoutScreen.tsx

```diff
@@ -241,7 +241,12 @@ const CheckoutScreen = (props: any) => {
                 merchantName: appPaymentParameters.merchantName || "Merchant name",
                 isCaptureBillingAddressEnabled: true,
                 showApplePayForUnsupportedDevice: true,
-                checkProvidedNetworks: false
+                checkProvidedNetworks: false,
+                shippingOptions: {
```

Comment:
It would be best if these were configurable - so it can be tested. iOS Debug App doesnt offer full control (we can enhance this, ran out of time), but it is possible to turn it on and off.
In the iOS Debug App, when `Capture Shipping Details` switch is on, we pass the following ShippingOptions:

```
{
  isCaptureShippingAddressEnabled: true,
  requireShippingMethod: true,
  additionalShippingContactFields: ['name', 'emailAddress', 'phoneNumber']
}
```

When the switch is off - we send `nil`. This default is somewhere in the middle, with false, false and some additional fields. 

**If making it configurable is too much work right now - I would make the default `nil` rather than this state.**